base/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+description: Base Helm chart for HMCTS Apps
+name: base
+# This is the chart version. This doesn't need to be incremented on every change.
+# It is managed by release pipeline based on Github tag
+version: 0.1.0 # Do not change
+appVersion: 0.1.0 # Do not change
+keywords:
+  - node
+  - javascript
+  - nodejs
+sources:
+  - https://github.com/hmcts/chart-base
+icon: https://nodejs.org/static/images/logos/nodejs-new-pantone-black.png
+dependencies:
+  - name: library
+    version: 0.4.3
```

Comment:
bit annoying that we have to duplicate the dependencies but i guess it's for the best? unless anyone has better ideas? we could just maintain one of these files and run a sed script for each chart?